src/managers/TileManager.ts

```diff
@@ -47,6 +47,17 @@ class TileManager {
             this.onSync.emit();
         });
     }
+    //mockfunction
+    public getTilesInHand(): Tile[] {
+        const allTiles: Nullable<Tile[]> = this.getAll();
+        assert(allTiles);
+        const nrTiles: int = 6;
```

Comment:
Does game rule say we draw first then play or first play then draw? I assume this is the number of tiles after we draw.

Reply:
We first play, then draw. But this function is only to mock/placehold the starting hand so she could implement her own code. This mock functions gets replaced by my logic later on.

Reply:
(This is only a placeholder for @Patertuck s method and should be removed before merging.)